include/zenoh-pico/api/primitives.h

```diff
@@ -97,6 +97,18 @@ z_owned_str_t z_keyexpr_to_string(z_keyexpr_t keyexpr);
  */
 z_bytes_t z_keyexpr_as_bytes(z_keyexpr_t keyexpr);
 
+/**
+ * Indicates if the key expression has been declared but don't guarantee it's still in session.
```

Comment:
I would add that it's not possible to get key expression string representation if it's declared and pointed to `zp_resolve`. This will make clear why this function is needed and connect this function to two functions above